src/test/java/uk/gov/hmcts/cmc/claimstore/services/notifications/CCJRequestedNotificationServiceTest.java

```diff
@@ -0,0 +1,96 @@
+package uk.gov.hmcts.cmc.claimstore.services.notifications;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mock;
+import org.mockito.junit.MockitoJUnitRunner;
+import uk.gov.hmcts.cmc.claimstore.controllers.utils.sampledata.SampleClaim;
+import uk.gov.hmcts.cmc.claimstore.controllers.utils.sampledata.SampleClaimData;
+import uk.gov.hmcts.cmc.claimstore.exceptions.NotificationException;
+import uk.gov.hmcts.cmc.claimstore.models.Claim;
+import uk.gov.hmcts.cmc.claimstore.models.sampledata.SampleTheirDetails;
+import uk.gov.service.notify.NotificationClient;
+
+import java.util.Arrays;
+
+import static org.mockito.ArgumentMatchers.anyMap;
+import static org.mockito.ArgumentMatchers.eq;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+import static uk.gov.hmcts.cmc.claimstore.utils.VerificationModeUtils.once;
+
+@RunWith(MockitoJUnitRunner.class)
+public class CCJRequestedNotificationServiceTest extends BaseNotificationServiceTest {
+
+    @Mock
+    private NotificationClient notificationClient;
+
+    private CCJRequestedNotificationService ccjRequestedNotificationService;
+
+    @Before
+    public void setup() {
+        ccjRequestedNotificationService = new CCJRequestedNotificationService(
+            notificationClient,
+            properties
+        );
+
+        when(templates.getEmail()).thenReturn(emailTemplates);
+        when(properties.getTemplates()).thenReturn(templates);
+        when(emailTemplates.getClaimantCCJRequested()).thenReturn(CLAIMANT_CCJ_REQUESTED_TEMPLATE);
+        when(emailTemplates.getDefendantCCJRequested()).thenReturn(DEFENDANT_CCJ_REQUESTED_TEMPLATE);
+    }
+
+    @Test
+    public void notifyClaimantShouldCallNotify() throws Exception {
+        Claim claim = SampleClaim.getDefault();
+
+        ccjRequestedNotificationService.notifyClaimant(claim);
+
+        verify(notificationClient, once())
```

Comment:
it shouldn't care how many times it is called.

Reply:
ok